core/storage/suggestion/gae_models_test.py

```diff
@@ -668,6 +668,147 @@ def test_get_question_suggestions_waiting_longest_for_review_max_fetch(
+            .get_number_of_translation_suggestions_in_review_in_lang_code(
+                self.translation_language_code
+            )
+        )
+
+        self.assertEqual(number_of_translation_suggestions, 2)
+
+    def test_get_num_translation_suggestions_in_review_in_lang_code_not_review(
+            self):
+        # A suggestion that has already been accepted.
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_ACCEPTED, 'author_3',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread1', self.translation_language_code)
+        # A suggestion that has already been rejected.
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp2', self.target_version_at_submission,
+            suggestion_models.STATUS_REJECTED, 'author_3',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp2.thread1', self.translation_language_code)
+
+        number_of_translation_suggestions = (
+            suggestion_models.GeneralSuggestionModel
+            .get_number_of_translation_suggestions_in_review_in_lang_code(
+                self.translation_language_code
+            )
+        )
+
+        # We're only looking for suggestions that are in review.
+        self.assertEqual(number_of_translation_suggestions, 0)
+
+    def test_get_num_translation_suggestions_in_review_in_lang_code_diff_lang(
+            self):
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_3',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread1', 'fr')
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp2', self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_3',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp2.thread1', 'fr')
+
+        number_of_translation_suggestions = (
+            suggestion_models.GeneralSuggestionModel
+            .get_number_of_translation_suggestions_in_review_in_lang_code('en')
+        )
+
+        self.assertEqual(number_of_translation_suggestions, 0)
+
+    def test_get_num_question_suggestions_in_review_returns_zero_if_not_review(
```

Comment:
You could also test that unrelated suggestions aren't returned even if all other criteria (e.g. lang/type) are matching (2 potential new tests).